packages/web-sockets/src/drivers/wb/server.js

```diff
@@ -0,0 +1,21 @@
+// External dependencies
+const { Server } = require('socket.io');
+
+// Internal dependencies
+const { Logger } = require('@booking-services/shared');
+
+const io = new Server();
+
+async function start() {
+  try {
+    io.listen(process.env.SERVER_PORT || 3000);
```

Comment:
I think we should not consume the env vars directly in the files, we can create a config file that consumes the env vars and verify if the format is the correct one, so we can consume config object props

Reply:
The default values should exist in that config file as well.